docs/architecture.md

```diff
@@ -2,20 +2,15 @@
 
 _Some incomplete notes_
 
-## Components
+![Lighthouse Architecture](https://raw.githubusercontent.com/GoogleChrome/lighthouse/master/assets/architecture.jpg)
 
-* **Driver** - Interfaces with [Chrome Debugging Protocol](https://developer.chrome.com/devtools/docs/debugger-protocol)  ([API viewer](https://chromedevtools.github.io/debugger-protocol-viewer/))
-* **Gatherers** - Requesting data from the browser (and maybe post-processing)
-* **Artifacts** - The output of gatherers
-* **Audits** - Non-performance evaluations of capabilities and issues. Includes a raw value and score of that value.
-* **Metrics** - Performance metrics summarizing the UX
-* **Aggregations** - Pulling audit results, grouping into user-facing components (eg. `install_to_homescreen`) and applying weighting and overall scoring.
-
-### Internal module graph
-
-![graph of lighthouse-core module dependencies](https://cloud.githubusercontent.com/assets/39191/19367685/04d4336a-9151-11e6-9ebb-3b87bdb09a4c.png)
+## Components & Terminology
 
-`npm install -g js-vd; vd --exclude "node_modules|third_party|fs|path|url|log" lighthouse-core/ > graph.html`
+* **Driver** - Interfaces with [Chrome Debugging Protocol](https://developer.chrome.com/devtools/docs/debugger-protocol)  ([API viewer](https://chromedevtools.github.io/debugger-protocol-viewer/))
+* **Gatherers** - Requests data from the browser or collects information from the page (and maybe post-processing)
+  * **Artifacts** - output of a gatherer
+* **Audits** - Tests for something. Either a pass/fail check or calculates a score.
```

Comment:
"Using the Artifacts as input, Audits evaluate a test and assign pass/fail/scoring."

Done